notebooks/search/03-ELSER.ipynb

```diff
@@ -75,7 +76,13 @@
     "``` -->\n",
     "<!-- # deploy model\n",
     "```json\n",
-    "POST _ml/trained_models/.elser_model_1/deployment/_start?deployment_id=for_search -->"
+    "POST _ml/trained_models/.elser_model_2/deployment/_start?deployment_id=for_search -->\n",
+    "\n",
+    "## Upgrading existing index to `.elser_model_2` ELSER model\n",
+    "\n",
+    "If have already had set up an index with ELSER model `.elser_model_1`, You can easily upgrade to  `.elser_model_2`  model by [Reindexing with ingest pipeline](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html#reindex-with-an-ingest-pipeline) referencing ELSER `.elser_model_2` and starting querying your data with ELSER right away. \n",
```

Comment:
We can also reference [update by query](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-update-by-query.html) as a way to update indices in place. We might want to do something like:

```suggestion
    "If have already had set up an index with ELSER model `.elser_model_1`, You can easily upgrade to  `.elser_model_2`  model by\n:"
    " -  [Reindexing with ingest pipeline](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html#reindex-with-an-ingest-pipeline) to ingest into a separate index\n"
    " - [[Update by query](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-update-by-query.html)] using an ingest pipeline to update the index in place\n"
    "In both cases, you will need to reference ELSER `.elser_model_2`.\n"
     "After reindexing or updating, you can start querying your data with ELSER right away. \n",
```

Reply:
Thanks @carlosdelest I added a note in [upgrading-index-to-use-elser notebook](https://github.com/elastic/elasticsearch-labs/blob/bc36bfa5d1ce56dc1e7412bc8a91db5ba1c36751/notebooks/model-upgrades/upgrading-index-to-use-elser.ipynb)
Commit bc36bfa5d1ce56dc1e7412bc8a91db5ba1c36751